.github/workflows/trymerge.yml

```diff
@@ -75,6 +75,7 @@ jobs:
           PR_NUM: ${{ github.event.client_payload.pr_num }}
         run: |
           set -ex
+          python3 .github/scripts/on_cancel_merge.py "${PR_NUM}"
           python3 .github/scripts/comment_on_pr.py "${PR_NUM}" "merge"
```

Comment:
Can we merge those scripts together? (If `comment_on_pr.py` already imports GitHub class

Reply:
I was originally going to do that, however, comment on pr is used in other bits which are not related to merging.